api/CloseEvent.json

```diff
@@ -0,0 +1,147 @@
+{
+  "api": {
+    "CloseEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CloseEvent",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "8",
+            "notes": "Prior to Firefox 8, the WebSocket close event was sent to the listener as a simple event. Prior to Firefox 12, the close code CLOSE_NORMAL was used when the channel was closed due to an unexpected error or unspecified error conditoin."
+          },
+          "firefox_android": {
+            "version_added": "8"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "CloseEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CloseEvent/CloseEvent",
+          "description": "<code>CloseEvent()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "8"
+            },
+            "firefox_android": {
+              "version_added": "8"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
```

Comment:
Is there a reason there's no status information here?

Reply:
Hm, I don't see any compat data for Safari on MDN, except for the `false` regarding `initCloseEvent`. Are you aware of other sources we could get this information from?

Reply:
Sorry for the confusion. This wasn't a gripe about the Safari data. I meant this is a question about the `"status"` block (i.e., whether this is experimental, standard, etc.)

Reply:
Note to myself: The consistency checker (#2463) could check that if a parent feature has a status block, the child feature must also have a status block (and must be consistent: if the parent feature is experimental/deprecated, the child feature must also be experimental/deprecated; if the parent feature is not on a standard-track, the child feature must not be on a standard-track either).

Reply:
@caugner while this is in progress, is it OK if I continue to review other PRs from you? I don't want to bombard you with notifications, if that's going to be too noisy for you

Reply:
@ddbeck Sure, that's fine. (My only problem really is "reading" notifications on the go and not being able to mark them as "unread" again, which is the reason I didn't react here.)

Reply:
@caugner Yeah, I have the same problem! I get email notifications too, but that's it's own set of headaches

Reply:
@ddbeck have added the status now. :wink: